apps/crn-frontend/src/tags/__tests__/Routes.test.tsx

```diff
@@ -101,8 +103,9 @@ it('Will search algolia using selected tag', async () => {
   userEvent.click(screen.getByRole('textbox'));
   userEvent.click(screen.getByText('LGW'));
   await waitFor(() =>
-    expect(mockGetResearchOutputs).toHaveBeenCalledWith(
+    expect(mockGetTagSearch).toHaveBeenCalledWith(
       expect.anything(),
+      ['research-output', 'user'],
```

Comment:
This seems like it will be anoying to update for each entity type. Could we test it once and then set the rest to anything?